tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
```

Comment:
I see - fyi you can actually retrieve idx from the node: `self.idx(node)` and the node from the idx: `self.get_node(idx)`

Since these are more or less interchangeable, I'd lean toward making the method signatures consistent